command/git_push_test.go

```diff
@@ -90,43 +88,6 @@ func TestGitPush(t *testing.T) {
 				assert.Equal(t, splitCommand, args)
 			}
 		},
-		"PushPatchMock": func(*testing.T) {
```

Comment:
this test still seems valid to me

Reply:
Whoops sorry, I meant to get rid of the other PushPatch test

Reply:
Sorry I was looking at the wrong thing with your other comment!